src/logging.js

```diff
@@ -1,17 +1,35 @@
-function logging(options = {}) {
-    const log = global.log = require('cnn-logger')(options);
+const debug = require('debug')('cnn-server:logger');
 
-    log.important('Initializing Logging subsystem...');
-    log.silly('Running self check: silly');
-    log.debug('Running self check: debug');
-    log.verbose('Running self check: verbose');
-    log.info('Running self check: info');
-    log.warn('Running self check: warn');
-    log.error('Running self check: error');
-    log.fatal('Running self check: fatal');
-    log.important('Running self check: important');
+function generateTags(LOGZIO_TAGS) {
+    if (!LOGZIO_TAGS) {
+        console.error('No LOGZIO_TAGS have been defined.');
```

Comment:
If LOGZIO_TAGS is not defined we could defaulted to process.env.PRODUCT + process.env.ENVIRONMENT (if defined) as you had before. In that we will always have logs available in logzio.

Reply:
we actually went around and around about this; having fallbacks in place if things were there, or weren't. but for simplicity and transparency we just went with a single env var.

if it's not set, it will display two messages to the consumer to add some tags. ENVIRONMENT and PRODUCT aren't used by all properties and there are other items that could be in the mix (e.g. edition).